src/testUtils/java/org/broadinstitute/hellbender/testutils/VariantContextTestUtils.java

```diff
@@ -302,7 +302,7 @@ public static void assertGenotypesAreEqual(final Genotype actual, final Genotype
 
     public static void assertGenotypesAreEqual(final Genotype actual, final Genotype expected, final List<String> extendedAttributesToIgnore) {
         Assert.assertEquals(actual.getSampleName(), expected.getSampleName(), "Genotype names");
-        Assert.assertTrue(CollectionUtils.isEqualCollection(actual.getAlleles(), expected.getAlleles()), "Genotype alleles");
+        Assert.assertTrue(CollectionUtils.isEqualCollection(actual.getAlleles(), expected.getAlleles()), "Genotype alleles: " + actual.getAlleles() + " != " + expected.getAlleles());
```

Comment:
This change came up while testing things but I figured it's useful so I'd keep it.